### DISCUSSION TITLE
Kurento installation on ubuntu 22.04

### DISCUSSION THREAD

**Discussion by devopsbugfeature** [Kurento/kurento#69] [Community]
Is there any supported kurento webrtc package available for ubuntu 22.04 version?

**Comment by ajaylashkari** [comment]
please update if there is any!??

**Reply by slabajo** [reply]
LAtest version is Kurento 7.1.0 that works on ubuntu noble (24.04). Debian packages and docker images are available as usual. Just follow installation guidelines on kurento.openvidu.io 

We have skipped Ubuntu 22.04 and went from kurento 7.0 on ubuntu 20.04 to kurento 7.1 on ubuntu 24.04

**Comment by devopsbugfeature** [comment]
Thank you!
